dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts

```diff
@@ -67,6 +74,7 @@ export class CustomPanelsAdaptor {
     try {
       const response = await client.callAsCurrentUser('observability.getObject', {
         objectType: 'operationalPanel',
+        maxItems: 10000,
```

Comment:
will this cause a performance issue? might be better to use pagination in the future

Reply:
Yes, this should be changed in future. For now I took the constant from [reporting](https://github.com/opensearch-project/dashboards-reports/blob/93d885c641c8c7fc8beeb081bee6ffd30b44f73b/dashboards-reports/server/routes/utils/constants.ts#L86-L87).